package.json

```diff
@@ -6,8 +6,9 @@
   "author": "Jon Wyatt <jonnywyatt@yahoo.co.uk>",
   "license": "MIT",
   "scripts": {
-    "build:logger": "babel ./logger --out-dir ./dist/logger --presets env,react",
-    "watch:logger": "babel ./logger --out-dir ./dist/logger --presets env,react --watch"
+    "build:iso-logger": "yarn build:client-logger && yarn build:server-logger",
+    "build:client-logger": "babel ./isomorphic-logger/src/clientLogger.js --out-file ./isomorphic-logger/client/index.js --presets env,react",
```

Comment:
should it output to a folder called dist? that's the convention used in other projects and clearly indicates those are compiled files that shouldn't be edited directly

Reply:
I choose this because this way the package can be imported this way: 
```
import ClientLogger from "module-shared-code/isomorphic-logger/client";
```
I think it produces much cleaner import statements. Also in the isomorphic-logger library, there is an src folder that also indicates that the source of the module comes from there.